app/views/components/error_summary.scala.html

```diff
@@ -26,7 +26,7 @@ <h2 class="h2-heading" id="error-summary-heading">
 
         <ul role="list" class="error-summary-list">
             @for(error <- errors) {
-                <li><a href="#@{error.key.replace(".", "_").replace('[', '_').replace("]", "")}">@messages(error.message, error.args:_*)</a></li>
+                <li><a id="@{error.key}-error" href="#@{error.key.replace(".", "_").replace('[', '_').replace("]", "")}">@messages(error.message, error.args:_*)</a></li>
```

Comment:
We should replace the `.`s before using the key in an ID

Reply:
You can create a variable with
```
@defining(error..key.replace(".", "_")) { id => 
  // some html
}
```

If you need to use it in the href too

Reply:
👍